ord_schema/units.py

```diff
@@ -66,8 +65,8 @@
     },
     reaction_pb2.Current: {
         reaction_pb2.Current.AMPERE: ['A', 'ampere', 'amps', 'amp'],
-        reaction_pb2.Current.MILLIAMPERE: ['mA', 'milliampere', 'milliamp',
-                                           'milliamps'],
+        reaction_pb2.Current.MILLIAMPERE:
+        ['mA', 'milliampere', 'milliamp', 'milliamps'],
```

Comment:
Do you mind using `INDENT_DICTIONARY_VALUE=True`?

https://github.com/google/yapf/issues/392
https://github.com/google/yapf/pull/483

Reply:
Done. Added the full config so we can tune as needed.